src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/geccovirologischerbefund/PCRObservationConverter.java

```diff
@@ -10,33 +10,45 @@
 import org.ehrbase.fhirbridge.ehr.opt.geccovirologischerbefundcomposition.definition.NachweisDefiningCode;
 import org.ehrbase.fhirbridge.ehr.opt.geccovirologischerbefundcomposition.definition.ProAnalytCluster;
 import org.ehrbase.fhirbridge.ehr.opt.geccovirologischerbefundcomposition.definition.VirusnachweistestDefiningCode;
+import org.hl7.fhir.r4.model.Identifier;
 import org.hl7.fhir.r4.model.Observation;
+import org.hl7.fhir.r4.model.Reference;
 
 import java.util.ArrayList;
 import java.util.List;
 
+@SuppressWarnings("java:S6212")
 public class PCRObservationConverter extends ObservationToObservationConverter<BefundObservation> {
 
     @Override
     protected BefundObservation convertInternal(Observation resource) {
         BefundObservation befundObservation = new BefundObservation();
-        befundObservation.setFeederAudit(createIdentifierSection(resource));
         befundObservation.setLabortestBezeichnungDefiningCode(LabortestBezeichnungDefiningCode.DETECTION_OF_VIRUS_PROCEDURE);
         befundObservation.setLabortestPanel(createLabortestPanel(resource));
+
+        if (resource.hasIdentifier()) {
+            befundObservation.setFeederAudit(createIdentifierSection(resource));
+        }
         return befundObservation;
     }
 
     private FeederAudit createIdentifierSection(Observation observation) {
         FeederAudit feederAudit = new FeederAudit();
```

Comment:
FeederAudit should be in generic not in specific ? 
Or is this a special case.

Reply:
I removed the FEEDER_AUDIT from the OBSERVATION as discussed on Slack